blacklight-spotlight.gemspec

```diff
@@ -24,7 +24,7 @@ these collections.)
   s.add_dependency 'acts-as-taggable-on', '>= 5.0', '< 7'
   s.add_dependency 'almond-rails', '>= 0.3.0'
   s.add_dependency 'autoprefixer-rails'
-  s.add_dependency 'blacklight', '~> 7.0'
+  s.add_dependency 'blacklight', '~> 7.12.1'
```

Comment:
Seeing an issue with:

```
Spotlight::Search#search_params maps the search to the appropriate facet values
     Failure/Error: expect(subject.search_params.to_hash).to include 'fq' => array_including('{!term f=genre_sim}map')

       expected {"f.corporate_name_ssm.facet.limit" => 11, "f.exhibit_exhibit-title-910_tags_ssim.facet.limit" => 11, "f....blic_bsi:false"], "hl.fl" => [], "qt" => "search", "rows" => 10, "sort" => "score desc, sort_title_ssi asc"} to include {"fq" => #<RSpec::Mocks::ArgumentMatchers::ArrayIncludingMatcher:0x000055b554eb0910 @expected=["{!term f=genre_sim}map"]>}
       Diff:
       @@ -1,14 +1,27 @@
       -"fq" => array_including({!term f=genre_sim}map),
       +"f.corporate_name_ssm.facet.limit" => 11,
       +"f.exhibit_exhibit-title-910_tags_ssim.facet.limit" => 11,
       +"f.genre_ssim.facet.limit" => 11,
       +"f.personal_name_ssm.facet.limit" => 11,
       +"facet" => false,
       +"facet.field" => ["genre_ssim", "personal_name_ssm", "corporate_name_ssm", "subject_geographic_ssim", "subject_temporal_ssim", "language_ssim", "exhibit_exhibit-title-910_tags_ssim"],
       +"facet.pivot" => [],
       +"facet.query" => [],
       +"fl" => "*",
       +"fq" => ["-exhibit_exhibit-title-910_public_bsi:false"],
       +"hl.fl" => [],
       +"qt" => "search",
       +"rows" => 10,
       +"sort" => "score desc, sort_title_ssi asc",
     # ./spec/models/spotlight/search_spec.rb:57:in `block (3 levels) in <top (required)>'
```

Reply:
I was going to say, I thought I fixed this already.. but I guess code you don't push up doesn't count (maybe I was going to think harder to see if it was a backwards compatibility concern?)

Basically -- change `genre_sim` (which doesn't exist and was probably a typo) to `genre_ssim`

Reply:
Got it thanks!